cli/src/pcluster/config/cluster_config.py

```diff
@@ -1405,6 +1406,10 @@ def __init__(
             shared_storage_type,
             default="Ebs",
         )
+        self.default_user_home = Resource.init_param(
+            default_user_home,
+            default="Shared",
```

Comment:
According to the changelog, the default user home is by default` /home`. Why is it `Shared` here?

Reply:
Clarified below: that value must be one of Shared or Local. 
What about defining an enumeration for this value?
The advantage would be self documentation: a developer could immediately jump to valid values.